lib/jquery.tinycarousel.js

```diff
@@ -143,19 +147,26 @@
         function _setEvents() {
             if(self.options.buttons) {
                 $prev.click(function() {
-                    self.move(--slideIndex);
+                    if(!$(this).hasClass("disable")) {
```

Comment:
I'm not sure if the original intention was to have the person implementing the class to handle the `disable` CSS class on their own. e.g., Check for disable and remove events if desired. It felt "right" to me to just do this in the plugin.